cli/CHANGELOG.md

```diff
@@ -9,3 +9,4 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Initial CLI is built and usable in it's most basic form. Listed endpoints can be scrolled.
+- Transaction Details can be seen
```

Comment:
Yikes, forgot about this.  Worry about it falling out of date if we add every endpoint to it, but if that's what we want to de we can be vigilant.